app/graphql/query/tripsCollection.graphql

```diff
@@ -0,0 +1,31 @@
+query tripsCollection($user_id: BigInt!) {
+  tripsCollection(filter: { user_id: { eq: $user_id } }) {
```

Comment:
Apparently `$user_id` has to do with PostgreSQL BIGSERIAL, but I'd say we'd rather expose UUID than it in terms of decoupling infras, security and whatnot.

Reply:
Totally agree about using UUIDs for better security. But researching pg_graphql will take time. Let's plan to tackle this later on. Sound good? 👀